lib/internal/bootstrap/loaders.js

```diff
@@ -97,6 +97,78 @@ const internalBindingWhitelist = new SafeSet([
   'zlib'
 ]);
 
+{
+  const { safeToString } = getInternalBinding('util');
+  const { setPrepareStackTraceCallback } = getInternalBinding('errors');
+  let wrapper;
+  const ErrorConstructor = Error;
+  setPrepareStackTraceCallback((globalThis, error, trace, message) => {
```

Comment:
@joyeecheung is this still the right place to set this up?

Reply:
Why not do this in node.js?
Also the implementation of the callback itself can be moved to some other modules e.g. per_thread.js, the only bits relavent to the bootstrap is the invocation to setPrepareStackTraceCallback